doc/man7/provider-keymgmt.pod

```diff
@@ -193,12 +207,41 @@ key object, but that is not mandatory.
 
 OP_keymgmt_free() should free the passed I<keydata>.
 
-The constructor and destructor are mandatory, a KEYMGMT implementation
-without them will not be accepted.
+OP_keymgmt_gen_init(), OP_keymgmt_gen_set_template(),
+OP_keymgmt_gen_set_params(), OP_keymgmt_gen_settable_params(),
+OP_keymgmt_gen() and OP_keymgmt_gen_cleanup() work together as a more
+elaborate context based key object constructor.
+
+OP_keymgmt_gen_init() should create the key object generation context
```

Comment:
The tense is wrong here.. 

so either change to 'OP_keymgmt_gen_init() creates'.... ,, determines

OR initialize

Reply:
No.  Please remember that these pages are primarily directed toward provider authors, so we're telling them what their implementations of these functions *should* do.  I've been trying to use a language similar to what's recommended RFC 2119.

Reply:
Interestingly, you don't seem to mind elsewhere...  so why did you pick out this function, specifically?

Reply:
Because you change the tense in the one sentence,

Reply:
Oh wait...  *next* line!  Ok.

Sorry for my blabbering.  Fixing